### PR TITLE
Ddl

### LINKED ISSUE #13: Create database

### LINKED ISSUE #14: Create database seed files

### PR DESCRIPTION
**Pull request by gabygab159**
Closes #13 
Closes #14 

All schema files created, seed files created, images stored to .docs.

### CODE REVIEW

File: db/schema/05_menu_item_categories.sql.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS widgets CASCADE;
```
Comment (wbox) [inline db/schema/05_menu_item_categories.sql.sql]: Should it be `DROP TABLE IF EXIST menu_item_categories CASCADE` ?

File: db/schema/07_orders.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
+DROP TABLE IF EXISTS orders CASCADE;
+
+CREATE TABLE orders (
+
+  id SERIAL PRIMARY KEY NOT NULL,
+  user_id INTEGER REFERENCES users(id) ON DELETE CASCADE,
+  restaurant_id INTEGER REFERENCES restaurants(id) ON DELETE CASCADE,
+  total DECIMAL(7,2) NOT NULL DEFAULT 0,
+  date DATE,
```
Comment (wbox) [inline db/schema/07_orders.sql]: Should we use `TIMESTAMP` instead `DATE`?

File: db/schema/08_messages.sql [hunk 3]
```diff
@@ -0,0 +1,10 @@
+DROP TABLE IF EXISTS messages CASCADE;
+
+CREATE TABLE messages (
+
+  id SERIAL PRIMARY KEY NOT NULL,
+  order_id INTEGER REFERENCES orders(id) ON DELETE CASCADE,
+  message TEXT,
+  date DATE
```
Comment (wbox) [inline db/schema/08_messages.sql]: Should we use `TIMESTAMP` instead `DATE`

File: db/seeds/01_users.sql [hunk 4]
```diff
@@ -1,3 +1,6 @@
 -- Users table seeds here (Example)
-INSERT INTO users (name) VALUES ('Alice');
-INSERT INTO users (name) VALUES ('Kira');
+INSERT INTO users (first_name, last_name, email, phone_number, password, address_id )
+VALUES ('Isaac', 'Asimov', 'iasimov@hotmail.com', 3212225456, 'foundation', 1),
+('Frank', 'Herbert', 'muadib@gmail.com', 5455557845, 'arakis', 2),
+('Ursula', 'Le Guin', 'ursula@gmail.com', 9698754254), 'lefthand', 3;
```
Comment (wbox) [inline db/seeds/01_users.sql]: Should we add our phone numbers while testing the app?

File: db/seeds/04_menu_items.sql [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+VALUES(1, 'Henryburger', 'a burger with cheese, lettuce, tomatoes', 'docs/menu_items_images/cheeseburger.jpg', 9, 5, 1),
+(1, 'Henryveggie', 'vegan friendly burger with cheese, lettuce, tomatoes', 'docs/menu_items_images/veggieburger.jpg', 8, 5, 1),
+(1, 'Henryfries', 'Henry approved french fries', 'docs/menu_items_images/fries.jpg', 3, 4, 2),
+(1, 'Poutine', 'french fries with cheese curds and gravy', 'docs/menu_items_images/poutine.jpg', 8, 6, 2 ),
+(1, 'House salad', 'lettuce and veggies - not Henry approved', 'docs/menu_items_images/salad.jpg', 7, 3, 2),
+(1, 'soda', 'bubbly and refreshing - not sponsored', 'docs/menu_items_images/soda.jpg', 2, 1, 3),
+(1, 'beer', 'bubbly and intoxicating', 'docs/menu_items_images/beer.jpg', 5, 1, 3),
+(1, 'bottled water', 'bottled fresh from the tap for your convenience', 'docs/menu_items_images/water.jpg', 3, 1, 3),
+(1, 'cupcake', 'end your meal with a cake that fits in a cup', 'docs/menu_items_images/cupcake.jpg',5, 1, 4),
+(1, 'doughnut', 'delicioius nut free doughnuts', 'docs/menu_items_images/doughnut.jpg', 4, 1, 4);
```
Comment (wbox) [inline db/seeds/04_menu_items.sql]: Should we have price as cents and convert them when showing on the page? Like the 8 dollars burger would be 800.

**Review by wbox — CHANGES_REQUESTED** [review]
Here are my suggestions:

1. Move all keys (primary and foreign) to the top of the tables
2. For same tables use the TIMESTAMP as type instead of DATE
3. Change the price to be represented as cents ($8 -> 800)

File: db/schema/05_menu_item_categories.sql.sql (hunk 1 above)
Comment (gabygab159) [reply db/schema/05_menu_item_categories.sql.sql]: good catch Sam! I will adjust it

File: db/schema/07_orders.sql (hunk 2 above)
Comment (gabygab159) [reply db/schema/07_orders.sql]: Will do! For some reason I thought the timestamp was on the query side.

**Pull request closed without merging**